contrib/node/examples/src/node/web-component-button/BUILD

```diff
@@ -29,3 +29,32 @@ node_bundle(
   name='web-component-button-bundle',
   node_module=':web-component-button'
 )
+
+# The following target invokes webpack build.
+node_module(
+  name='web-component-button-processed',
+  sources=globs('package.json', 'webpack.config.js', 'src/*', 'test/*'),
+  dependencies=[
+    'contrib/node/examples/3rdparty/node/mocha',
+    'contrib/node/examples/3rdparty/node/react',
+    'contrib/node/examples/src/node/web-build-tool',
+  ],
+  build_script='build',
+)
+
+node_bundle(
+  name='web-component-button-processed-bundle',
+  node_module=':web-component-button-processed'
+)
+
+node_module(
```

Comment:
Added a line of comments.  This target could be depending on `:web-component-button-processed`, but it will generate a different output structure.

current BUILD rule => 
```
JAR / web-component-button-processed-with-dependency-artifacts / Button.js
JAR / web-dependency-test/...
```

change BUILD rule to 
```
node_module( 
  name = 'module_name',
  dependencies=[':web-component-button-processed', 'web-dependency-test'])
```
will generate =>
```
JAR / web-component-button-processed / Button.js
JAR / web-dependency-test/ ...
JAR / module_name / node_modules / web-component-button-processed / %source_files%
JAR / module_name / node_modules / % other_node_modules % / ...
```
Basically, each module's artifacts will be included at JAR root separately, disregarding the hierarchy.